### DISCUSSION TITLE
How do I change the web gui password?

### DISCUSSION THREAD

**Discussion by warwertt123** [coder/code-server#5029] [Q&A]
I build code-server by using docker, may I change the password after building the container?
I follow the FAQ **"How do I change the password?"** 
to modify the password in config  (~/.config/code-server/config.yaml ), then restart code-server,
but when I log in by WEB, it doesn't work, the password doesn't change. 
What can I do to modify the password?
It must rebuild the container?

**My docker-compose.yaml**
![image](https://user-images.githubusercontent.com/46851653/160067895-9d68712c-1024-46e7-a1cf-c7b6dcface14.png)

**My config.yaml**
![image](https://user-images.githubusercontent.com/46851653/160063978-69b55051-e63c-4d0a-b980-96c9ce93fa52.png)

**Comment by jsjoeio** [comment]
Yes! I believe you'll need to rebuild the container to get it to take the new password 🤔 Let me know if that works

**Comment by warwertt123** [comment]
It works, thanks for your reply.
But is there any method to change the password manually without rebuilding the container?
Like edit a config file or use the shell script.
I want to build the container for users who can't use docker.

**Reply by jsjoeio** [reply]
Hmm...I think if you edit this file `~/.config/code-server/config.yaml` and then hard reload (clear cache), it should work. Can you try that?

**Reply by warwertt123** [reply]
Sorry....it doesn't work. I had already modified the password, cleared the cache, and restart the code-server, but the password is still the same, I install the code-server on ubuntu-20.04.4 by using docker-compose, don't know what I did is wrong. Is this method theoretically feasible?

**Reply by jsjoeio** [reply]
Hmm...I thought so. The password is hashed and then we store something in localStorage. Maybe it [doesn't get cleared](http://sharonminsuk.com/blog/2011/03/21/clearing-cache-has-no-effect-on-html5-localstorage-or-sessionstorage/). Can you try clearing localStorage and seeing if that works?
